app/components/verkiezingen/prepare-legislatuur-section.hbs

```diff
@@ -6,52 +6,76 @@
-      {{@bestuursorgaan.isTijdsspecialisatieVan.naam}}</AuHeading>
-    {{#unless @isBehandeld}}
-      <AuContent @skin="small" class="au-u-flex-self-end">
-        {{#if (or (await @bestuursorgaan.isGR) (await @bestuursorgaan.isCBS))}}
+  <div class="au-u-flex au-u-flex--column au-u-flex--row-when-medium">
+    <div
+      class="au-u-flex au-u-flex--between au-u-margin-left au-u-margin-right"
+    >
+      <AuHeading
+        @skin="3"
+        class="au-u-flex-self-start au-u-margin-bottom-small"
+      >Samenstelling
+        {{@bestuursorgaan.isTijdsspecialisatieVan.naam}}</AuHeading>
+    </div>
+    <div class="au-u-flex au-u-flex--end au-u-margin-left au-u-margin-right">
+      {{#unless @isBehandeld}}
+        <AuContent @skin="small" class="au-u-flex-self-end">
           {{! template-lint-disable no-inline-styles }}
           <AuButton
             style="background-color: #007A37; border-color: #007A37;"
+            class="au-u-margin-top-none"
             @icon="table-row-end-add"
             @loadingMessage="Genereren"
             {{on "click" (fn (mut this.isGeneratingRows) true)}}
           >
             Genereer rijen
           </AuButton>
-        {{/if}}
-        {{#if
-          (or
-            (await @bestuursorgaan.isRMW) (await @bestuursorgaan.isVastBureau)
-          )
-        }}
+
           <AuButton
-            @skin="secondary"
-            @icon="synchronize"
-            @loading={{this.mirrorTable.isRunning}}
-            @loadingMessage="Aan het synchroniseren"
-            {{on "click" (perform this.mirrorTable)}}
+            @icon="add"
+            @loading={{this.onCreate.isRunning}}
+            class="au-u-margin-top-none"
+            @loadingMessage="Aan het toevoegen"
+            {{on "click" this.createMandataris}}
           >
-            {{#if (await @bestuursorgaan.isRMW)}}
-              Neem over van Gemeenteraad
-            {{/if}}
-            {{#if (await @bestuursorgaan.isVastBureau)}}
-              Neem over van College van Burgemeester en Schepenen
-            {{/if}}
+            Voeg toe
           </AuButton>
-        {{/if}}
-        <AuButton
-          @icon="add"
-          @loading={{this.onCreate.isRunning}}
-          @loadingMessage="Aan het toevoegen"
-          {{on "click" this.createMandataris}}
-        >
-          Toevoegen
-        </AuButton>
-      </AuContent>
-    {{/unless}}
+          {{#if
+            (or
+              (await @bestuursorgaan.isRMW) (await @bestuursorgaan.isVastBureau)
+            )
+          }}
+            <div class="au-u-flex--inline">
+              <Shared::Tooltip
+                @showTooltip={{true}}
+                @tooltipText={{if
+                  (await @bestuursorgaan.isRMW)
+                  "Neem de mandatarissen over van de gemeenteraad"
+                  "Neem de mandatarissen over van het College van Burgemeester en Schepenen"
+                }}
+                @alignment="left"
```

Comment:
I would align the tooltip right, since it is the rightmost button, and that way the alignment looks a bit better in my opinion.

Reply:
alright!